.github/workflows/commit-message-check.yaml

```
@@ -22,9 +22,11 @@ jobs:
     - name: Get PR Commits
       if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
       id: 'get-pr-commits'
-      uses: tim-actions/get-pr-commits@v1.0.0
+      uses: tim-actions/get-pr-commits@v1.2.0
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
+        filter_out_pattern: '^\s*revert'
+        filter_out_flags: 'i'
```

I'm generally okay with the change, but I think we should restrict this to the format created by `git-revert`. I.e.
```suggestion
        filter_out_pattern: '^Revert '
```

---

done, I have changed the regex to '^Revert "'